src/edu/csus/ecs/pc2/clics/API202306/CLICSCommand.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * CLICS Command information
+ * Contains information about a command, such as a compiler or runner
+ * 
+ * @author John Buck
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CLICSCommand {
+
+    @JsonProperty
+    private String command;
+
+    @JsonProperty
+    private String args;
+
+    @JsonProperty
+    private String version;
+
+    @JsonProperty
+    private String version_command;
+
+    /**
+     * Fill in command information properties
+     * 
+     * @param command The command to execute
+     * @param args Arguments to command, null if none
+     */
+    public CLICSCommand(String command, String args) {
```

Comment:
The 2023-06 CLICS spec says that `version` defaults to the empty string, and `version_command` defaults to the string `<command> --version`.   However, neither this constructor nor the following one enforces that rule; the variables are both declared as uninitialized Strings, which I'm pretty sure means they are defaulting to `null` (which of course is not the same as "empty string").

Reply:
All the properties in a Command are optional in the spec, except command.  null properties are not part of the generated JSON. (`@JsonInclude(JsonInclude.Include.NON_NULL`).  The spec also says: "_The compiler and runner objects are intended for informational purposes. It is not expected that systems will synchronize compiler and runner settings via this interface._" So, i wasn't too awful concerned about what the strings are.  Should I be?

Reply:
I guess it's not that big a deal; it just seemed like something worth noting.  I'll resolve this comment based on the BFTF (Bigger Fish To Fry) approach :)